NumericInput/NumericInput.js

```diff
@@ -122,7 +122,7 @@ export default class NumericInput extends Component {
 
     render() {
         const editable = this.props.editable
-        const sepratorWidth = this.props.sepratorWidth
+        const sepratorWidth = this.props.sepratorWidth || this.props.sepratorWidth
```

Comment:
you again wrote sepratorWidth and not separatorWidth

Reply:
I donot know how to update the pull request so I am closing this one, and opening another.